src/mpc-hc/MainFrm.cpp

```diff
@@ -11189,6 +11189,10 @@ void CMainFrame::OpenSetupAudio()
 {
     m_pBA->put_Volume(m_wndToolBar.Volume);
 
+    if (AfxGetAppSettings().nCLSwitches & CLSW_MUTE) {

```

Comment:
Probably should be before put_Volume. Also `OpenSetupAudio()` is called when starting playback. I think it should set volume not only in this case. That say probably better place to set vol is when restoring setting. https://github.comhttps://github.com/mpc-hc/mpc-hc/blob/develop/src/mpc-hc/AppSettings.cpp#L760

(simply set fMute when parsing cmdline)